src/www/ui/api/documentation/openapi.yaml

```diff
@@ -797,6 +797,79 @@ paths:
                           "fossy"        
         default:
           $ref: '#/components/responses/defaultResponse'
+  /users/tokens:
+    post:
+      operationId: createRestApiToken
+      tags:
+        - User
+        - Admin
+      summary: Create a new REST API Token
+      description: >
+        Create a new REST API Token
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                token_name:
+                  type: string
+                token_scope:
+                  type: string
+                token_expire:
+                  type: string
+              required:
+                - token_name
+                - token_scope
+                - token_expire
+      responses:
+        '201':
+          description: Token created successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Info'
+        '400':
+          description: Following parameters are required in the request body -- token_name,token_scope,token_expire
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Info'
+        
+        default:
+            $ref: '#/components/responses/defaultResponse'
+  /users/tokens/{type}:
+    parameters:
+      - name: type
+        required: true
+        in: path
+        schema:
+          type: string
```

Comment:
Please define what are the allowed values of `type`.
Consider using [enum](https://spec.openapis.org/oas/v3.1.0#serverVariableEnum) type.

Reply:
Sure, I'll do the change!